public/app/features/plugins/sandbox/utils.ts

```diff
@@ -24,35 +19,22 @@ export function assertNever(x: never): never {
   throw new Error(`Unexpected object: ${x}. This should never happen.`);
 }
 
+const sandboxLogger = createMonitoringLogger('features.plugins.sandbox', { monitorOnly: String(monitorOnly) });
```

Comment:
All this feature monitoring is already based on the source being "sandbox" I prefer we keep it like that.